jobs/integr8ly/pds-install.yaml

```diff
@@ -139,6 +151,7 @@
                     stage('Prepare environment') {
                         dir('installation') {
                             sh """
+                                sudo oc login -u system:admin
```

Comment:
why this command is here? You only play with the files so no need to be logged in?

And this runs on bastion directly so I think you are logged in as system:admin by default, aren't you?

Reply:
this command was moved from below to be able to create a secret and ns (I believe it is still needed here)

Reply:
hmm, strange, but ok :)